tests/tracer/test_tracer.py

```diff
@@ -560,6 +560,21 @@ def test_tracer_shutdown_timeout():
     t._writer.stop.assert_called_once_with(timeout=2)
 
 
+def test_tracer_shutdown_warning():
+    t = ddtrace.Tracer()
+    t.shutdown()
+
+    with mock.patch.object(logging.Logger, "warning") as mock_logger:
+        with t.trace("something"):
+            pass
+
+    mock_logger.assert_has_calls(
+        [
+            mock.call("Spans started after tracer has been shut down will not sent to the Datadog Agent."),
```

Comment:
```suggestion
            mock.call("Spans started after the tracer has been shut down will not be sent to the Datadog Agent."),
```

Warning changed in a previous commit